ubiquity/dell-bootstrap.py

```diff
@@ -112,7 +122,8 @@ def __init__(self, controller, *args, **kwargs):
         self.efi = False
         with misc.raised_privileges():
             self.genuine = magic.check_vendor()
-
+        with misc.raised_privileges():
```

Comment:
Does this actually need elevated privs?  I thought that `/sys/class/dmi/id/product_sku` can be read by anyone by default.